state/runtime/evm/instructions.go

```diff
@@ -1403,6 +1414,48 @@ func opHalt(op OpCode) instruction {
 	}
 }
 
+func opAuth(c *state) {
+	c.authorized = nil
+
+	commit := c.pop()
```

Comment:
Is this aligned with the spec [here](https://eips.ethereum.org/EIPS/eip-3074#authcall-0xf7)? `AUTH` opcode has three values on the stack and they are memory pointers not the signature values.

Reply:
The current solidity compiler used by our smart contract team contains a patch that is only compatible with the previous revision of the specification. While it would be relatively easy for us to convert the code from the `bor` repository (https://github.com/maticnetwork/bor/commit/54df42f550d57b27423ed178f7469b011ac609b7) into `edge` we are facing issues with the smart contract side of the conversion